tracker/tracker/src/main/app/index.ts

```diff
@@ -105,11 +109,13 @@ export default class App {
         verbose: false,
         __is_snippet: false,
         __debug_report_edp: null,
+        localStorage: window.localStorage,
+        sessionStorage: window.sessionStorage,
       },
       options,
     );
     if (sessionToken != null) {
-      sessionStorage.setItem(this.options.session_token_key, sessionToken);
+      this.sessionStorage.setItem(this.options.session_token_key, sessionToken);
```

Comment:
sessionStorage being called before being assigned in line 129, which would break in your use case